docs/frontend/setup/run-storybook.md

```diff
@@ -48,6 +48,24 @@ storybook_1  | │                                                 │
 storybook_1  | ╰─────────────────────────────────────────────────╯
 ```
 
+### Option C: Viewing a Pull Request artifact
+
+If you don't want to set up any development tools on your local machine, another option is to use the static copy of storybook which is automatically generated as part of every pull request created by developers, although finding this does require a little bit of work.
+
+**Step 1**: When viewing a pull request on GitHub, scroll down to the section of the page that talks about automated checks. (This is also where you'll find the link to review and approve Happo diffs.) Scroll to the line with the title `ci/circleci: build_storybook`, and click on the "Details" link.
+
+![build_storybook check](/img/pr_storybook/github_check.png)
+
+**Step 2**: On the CircleCI page, you'll see several tabs near the middle of the page. Click on "Artifacts."
```

Comment:
I might call out here that they may need to login to CircleCI using their GitHub account if they have not previously done so or have been logged out.

Reply:
Updated with instructions to log in